### PR TITLE
gh #212: dsFPD L1 issuefix 

### PR DESCRIPTION
**Pull request by Sumip98**
Update the loop condition in test_l1_dsFPD_negative_dsGetFPState

### CODE REVIEW

File: src/test_l1_dsFPD.c [hunk 1]
```diff
@@ -813,7 +813,6 @@ void test_l1_dsFPD_positive_dsGetFPBrightness (void)
     uint8_t count = 0;
     dsFPDIndicator_t eIndicator;
     char buffer[DS_FPD_KEY_SIZE];
-    int indicator_power_index = -1;
```
Comment (akhilbhas) [inline src/test_l1_dsFPD.c:816]: Unused variable removed ?

Comment (Sumip98) [reply src/test_l1_dsFPD.c:816]: After building FPD, an unused variable 'indicator_power_index' was identified in test_l1_dsFPD_positive_dsGetFPBrightness

Comment (akhilbhas) [reply src/test_l1_dsFPD.c:816]: oK

**Review by akhilbhas — APPROVED** [review]
Looks Good

**Review by hari22yuva — APPROVED** [review]
ship it

**Pull request merged**